ochami-cli

```diff
@@ -457,7 +458,7 @@ def setArgs():
     subparser = parser.add_subparsers(dest='command')
 
     smd = subparser.add_parser('smd')
-    smd.add_argument('--config', type=str)
+    smd.add_argument('--config', type=str, help="Path to YAML-formatted payload to send to SMD")
```

Comment:
I agree. `--config` seems to imply that there is a CLI config file being used. I think it would be good to implement this in a separate PR in order to maintain the scope of this one.